cmd/promtool/tsdb.go

```diff
@@ -419,6 +421,64 @@ func openBlock(path, blockID string) (*tsdb.DBReadOnly, tsdb.BlockReader, error)
 	return db, block, nil
 }
 
+func relabelBlock(path, blockID, file string, addChangelog bool) error {
+	db, block, err := openBlock(path, blockID)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		err = tsdb_errors.NewMulti(err, db.Close()).Err()
```

Comment:
it seems like a named return would be clearer?

Reply:
Done